src/api/app/controllers/webui/projects/meta_controller.rb

```diff
@@ -5,6 +5,16 @@ class MetaController < WebuiController
       before_action :validate_meta, only: [:update], if: -> { params[:meta] }
       after_action :verify_authorized, only: [:update]
 
+      rescue_from Project::Errors::UnknownObjectError do |exception|
+        message = exception.message || exception.default_message
+        if request.xhr?
```

Comment:
We do javascript requests to this controller?

Reply:
Yes, to update the content.